book/coloring_and_theming.md

```diff
@@ -34,21 +28,27 @@ Here are the current options for `$env.config.table.mode`:
 - `none`
 - `other`
 
-### Color Symbologies
+## Color Configuration
+
+The color configuration is defined in `$env.config.color_config`. The current configuration can be printed with:
+
+```nu
+$env.config.color_config | sort
+```
 
----
+The color and style-attributes can be declared in multiple alternative formats.
 
 - `r` - normal color red's abbreviation
 - `rb` - normal color red's abbreviation with bold attribute
 - `red` - normal color red
 - `red_bold` - normal color red with bold attribute
 - `"#ff0000"` - "#hex" format foreground color red (quotes are required)
 - `{ fg: "#ff0000" bg: "#0000ff" attr: b }` - "full #hex" format foreground red in "#hex" format with a background of blue in "#hex" format with an attribute of bold abbreviated.
+- `{|x| 'yellow' }` - closure returning a string with one of the color representations listed above
+- `{|x| { fg: "#ff0000" bg: "#0000ff" attr: b } }` - closure returning a valid record
 
 ### Attributes
 
----
-
```

Comment:
The list of color "code" and "name" is incomplete. You can see the complete list with `ansi --list | first 133`

Reply:
https://github.com/nushell/nushell.github.io/pull/1775